component/Users/assets/css/main.css

```diff
@@ -0,0 +1,64 @@
+#wpadminbar .quicklinks #wp-admin-bar-tikemp_impresonate_user ul li .ab-item {
```

Comment:
At some point we should use SASS in our plugins?

Reply:
We should. I just copied this from the prev plugin but we should look at putting sass into components plugin